### ISSUE TITLE
Group Movies into Collections

### ISSUE THREAD

**Issue by Niach** [dularion/streama#528]
It would be nice if the collections from tmdb are used, so you can group trilogies etc. into one entry in the overview, which opens a series-like modal.

**Comment by Thovi98** [comment]
I agree with this ! It would correct the issue that when alphabetic order is set, the different movies from a trilogy are not grouped together and in right order (first movie first, and then in chronological order) : for now it’s also in alphabetic order, but it’s not so logical for me...
